sirepo/template/webcon.py

```diff
@@ -412,6 +515,94 @@ def write_epics_values(server_address, fields, values):
     return True
 
 
+def _add_cells_cluster(notebook, data_var, fields, col_limit, x_var, y_var,
+        x_label, y_label, rpt):
+    notebook.add_code_cell('from sirepo.analysis import ml')
```

Comment:
Part of this notebook is promoting good style. Importing `ml` is bad style, because it's too hard to know where it comes from in the places it is used. Below it's used correctly `sirepo.analysis.ml` so don't import `ml`, just use `sirepo.analysis.ml`, but actually, you'll implement #2939 and use `sirepo.lib.JupyterSupport('webcon')`.